paper/paper.md

```diff
@@ -0,0 +1,150 @@
+
+Finally, `mpl-interactions` allows the reuse of Python functions performing mathematical operations in multiple parts of user code, rather than requiring users to write a plotting specific version. For example, below the `logistic_growth` function is used for both curve fitting and an interactive display in order to better understand the role of the model parameters.
+
+```python
+%matplotlib ipympl
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.optimize import curve_fit
+
+import mpl_interactions.ipyplot as iplt
+
+
+def logistic_growth(t, L, k, t0):
+    return L / (1 + np.exp(-k * (t - t0)))
+
+
+# create a synthetic dataset of logistic growth
+rng = np.random.default_rng(seed=1995)
+t_data = np.sort(rng.uniform(0, 10, size=50))
+y_data = logistic_growth(t_data, L=5, k=1, t0=1) + rng.normal(size=t_data.size, scale=0.1)
+
+
+# You can use the `logistic_growth` function to curve_fit
+popt, pcov = curve_fit(logistic_growth, t_data, y_data)
+
+
+# Now you can directly the same function to make
+# an interactive plot to better understand its parameters
+fig, axs = plt.subplots(1, 2, sharey=True)
+axs[0].plot(t_data, y_data, "o")
+axs[0].plot(t_data, logistic_growth(t_data, *popt))
+axs[0].set_title("Data + Fit")
+axs[1].set_title("Interactive Exploration")
+
+ctrls = iplt.plot(
+    np.linspace(0, 10),
+    logistic_growth,
+    L=(0.5, 10),
+    k=(0.1, 1),
+    t0=(0, 2.5),
+    ax=axs[1],
+    label="interactive",
+)
+axs[1].plot(t_data, logistic_growth(t_data, *popt), "--", label="fit")
+plt.legend()
+```
+
+![Generated figure and sliders after running above example in jupyter lab.\label{fig:logistic}](imgs/logistic_growth-dark.png){ width=75% }
+
+This framework makes it easy generate complex interactive visualizations. It also enables `mpl-interactions` to manage generating GIFs. Any parameter controlled through `mpl-interactions` can be used to automatically generate a gif of the plot changing as a function of that parameter ([Animation Documentation](https://mpl-interactions.readthedocs.io/en/stable/examples/animations.html)). Thus `mpl-interactions` can assist across the data visualization process, from initial exploration to the creation of a final animated plot as a GIF.
+
+# Acknowledgements
+
+WThis work was supported by a National Defense Science and Engineering Graduate Fellowship (FA9550-19-F-0008, to IHI), the George W. Merck Fund of the New York Community Trust (award 338034, to DRH), and funds from Harvard University.
+
+We thank Dr. K. Dalton from stimulating discussion, and Easun Arunachalam for feedback on drafts of this paper. In addition, many users have contributed features and bug fixes. Of particular note are Remco de Boer, John Russell, and Samantha Hamilton who made contributions to documentation and code, code, and documentation respectively. A full list of coding contributors can be found here: https://github.com/mpl-extensions/mpl-interactions/graphs/contributors
```

Comment:
attention: @redeboer @samanthahamilton I hope you don't mind getting a shout out here :)

The other people I know in real life so will check with them offline